src/main/scala/org/broadinstitute/dsde/firecloud/webservice/LibraryApiService.scala

```diff
@@ -0,0 +1,65 @@
+package org.broadinstitute.dsde.firecloud.webservice
+
+import org.broadinstitute.dsde.firecloud.FireCloudConfig
+import org.broadinstitute.dsde.firecloud.model.ModelJsonProtocol.impCurator
+import org.broadinstitute.dsde.firecloud.model.{Curator, UserInfo}
+import org.broadinstitute.dsde.firecloud.service.{FireCloudDirectives, FireCloudRequestBuilding, LibraryService}
+import org.broadinstitute.dsde.firecloud.utils.StandardUserInfoDirectives
+import spray.client.pipelining._
+import spray.http.StatusCodes._
+import spray.httpx.SprayJsonSupport._
+import spray.json._
+import spray.routing._
+
+import scala.concurrent.ExecutionContext
+
+trait LibraryApiService extends HttpService with FireCloudRequestBuilding
+  with FireCloudDirectives with StandardUserInfoDirectives {
+
+  private implicit val ec: ExecutionContext = actorRefFactory.dispatcher
+
+  lazy val rawlsCuratorUrl = FireCloudConfig.Rawls.authUrl + "/user/role/curator"
+
+  val libraryServiceConstructor: UserInfo => LibraryService
+
+  val libraryRoutes: Route =
+    pathPrefix("schemas") {
+      path("library-attributedefinitions-v1") {
+        respondWithJSON {
+          withResourceFileContents("library/attribute-definitions.json") { jsonContents =>
+            complete(OK, jsonContents)
+          }
+        }
+      }
+    } ~
+    pathPrefix("api") {
```

Comment:
Interesting choice to pull out api vs other root level calls here. Current practice is to handle api/non-api at the application level so we can apply the necessary authentication protocols to those prefixed calls.